ios/AirMaps/AIRMapMarker.m

```diff
@@ -223,5 +223,13 @@ - (void)setImageSrc:(NSString *)imageSrc
                                                           }];
 }
 
+- (void)setPinColor:(UIColor *)pinColor
+{
+    _pinColor = pinColor;
+    
+    if ([_pinView respondsToSelector:@selector(setPinTintColor:)]) {
```

Comment:
do we need to check if `_pinView` is nil here before we can call `respondsToSelector:` on it?